conversational-skills/README.md

```diff
@@ -0,0 +1,304 @@
+# Pro Code Conversational Skills: Registering a Pro-Code Conversational Skill Provider
+
+The following document explains how a watsonx Assistant client should **invoke** Assistant's APIs in order to enable them to use the pro-code conversational skill feature. 
+
+## Table of Contents
+
+- [Overview of Conversational Skills](#Overview) 
+- [Register a Conversational Skill Provider](#Register-a-Conversational-Skill-Provider) | `POST /v2/providers`
+- [Retrieve Conversational Skill Provider(s)](#Retrieve-Conversational-Skill-Providers) | `GET /v2/providers`
+- [Update a Conversational Skill Provider](#Update-a-Conversational-Skill-Provider) | `POST /v2/providers/{{provider_id}}`
+- [Create a Skill-Backed Action](#Create-a-Skill-Backed-Action) | `POST /v2/assistants/{{assistant_id}}/skills/{{action_skill_id}}`
+
+### Overview
+
+Skill-backed actions, or _conversational skill actions_, enable builders of watsonx Assistant to begin tasks given their users' conversation input. Each task and its specification is fully contained in a conversational skill. Builders may choose to create skills through watsonx Orchestrate's skill builder, or choose to manage more complex workflows offline by independently creating / maintaining their own skills. 
+
+Pro-code creators of conversational skills must do the following: 
+(1) Register a conversational skill provider
+(2) Implement the [client-side endpoints](procode-endpoints.md) to the written specification on their personal host
+(3) Create a skill-backed action either through the API or through the watsonx Assistant UI*. 
```

Comment:
```suggestion
(3) Create a skill-backed action either through the API or through the watsonx Assistant UI. 
```

Reply:
I want to keep this * because it's a reference to a callout at line 23